example/XmlExample.php

```diff
@@ -0,0 +1,366 @@
+<?php
```

Comment:
* In case we want to have more examples in the future, e.g. with CSV, rename the directory to `examples`.
* Reference the examples directory in `README.md`

Reply:
Done